src/openrct2/ride/thrill/meta/SwingingShip.h

```diff
@@ -23,5 +23,6 @@ constexpr const RideTypeDescriptor SwingingShipRTD =
                      RIDE_TYPE_FLAG_VEHICLE_IS_INTEGRAL | RIDE_TYPE_FLAG_TRACK_NO_WALLS | RIDE_TYPE_FLAG_FLAT_RIDE |
                      RIDE_TYPE_FLAG_HAS_VEHICLE_COLOURS | RIDE_TYPE_FLAG_ALLOW_MUSIC | RIDE_TYPE_FLAG_HAS_ENTRANCE_EXIT |
                      RIDE_TYPE_FLAG_SINGLE_SESSION | RIDE_TYPE_FLAG_INTERESTING_TO_LOOK_AT),
+    SET_FIELD(NameConvention, { RIDE_COMPONENT_TYPE_SHIP, RIDE_COMPONENT_TYPE_STRUCTURE, RIDE_COMPONENT_TYPE_STATION }),
```

Comment:
Is this the `// 1a Pirate Ship` mentioned on `RideData`? (Seems ok if it is, just wanted to confirm)

Reply:
Yup. Also see the track paint function field above. We renamed it, but didn't replace all references.

Also see the the RIDE_TYPE_* enums.

Reply:
Ok, thanks!